content/css-selectors/attribute-selectors.md

```diff
@@ -0,0 +1,53 @@
+# Selectors By Attribute
+
+## Description
+
+CSS has the ability to target HTML elements based on any one of their attributes. An attribute selector will match elements on the basis of either the presence of an attribute, or the exact or partial match of an attribute value. Attribute selectors were introduced in CSS2, and CSS3 added a few more.
+
+Attribute selectors are delimited by square brackets; the simplest form of an attribute selector consists of an attribute name surrounded by square brackets for example:
+
+	tag[attribute] {
```

Comment:
You can use fenced blocks to render the code here https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks